client/settings/payment-settings/promotional-banner/stripe-tax-banner.js

```diff
@@ -0,0 +1,109 @@
+import { React } from 'react';
+import styled from '@emotion/styled';
+import interpolateComponents from '@automattic/interpolate-components';
+import { __ } from '@wordpress/i18n';
+import apiFetch from '@wordpress/api-fetch';
+import CardBody from 'wcstripe/settings/card-body';
+import illustration from 'wcstripe/settings/payment-settings/promotional-banner/illustrations/stripe-tax.svg';
+import {
+	BannerIllustration,
+	ButtonsRow,
+	CardColumn,
+	CardInner,
+	DismissButton,
+	MainCTALink,
+} from 'wcstripe/settings/payment-settings/promotional-banner/banner-layout';
+import { recordEvent } from 'wcstripe/tracking';
+import { ExternalLink } from '@wordpress/components';
```

Comment:
Non-blocking nit: Shouldn't these imports be in order?
```suggestion
import interpolateComponents from '@automattic/interpolate-components';
import styled from '@emotion/styled';
import apiFetch from '@wordpress/api-fetch';
import { ExternalLink } from '@wordpress/components';
import { __ } from '@wordpress/i18n';
import CardBody from 'wcstripe/settings/card-body';
import illustration from 'wcstripe/settings/payment-settings/promotional-banner/illustrations/stripe-tax.svg';
import {
	BannerIllustration,
	ButtonsRow,
	CardColumn,
	CardInner,
	DismissButton,
	MainCTALink,
} from 'wcstripe/settings/payment-settings/promotional-banner/banner-layout';
import { recordEvent } from 'wcstripe/tracking';
```